swc/private/swc.bzl

```diff
@@ -244,10 +245,14 @@ def _impl(ctx):
     else:
         output_sources = []
 
+        build_dir = paths.dirname(ctx.build_file_path)
+        root_dir = paths.join(build_dir, ctx.attr.root_dir)
+        root_dir_file = paths.join(root_dir, "x")
```

Comment:
nit: `x` maybe be a leftover? i am not sure what it meant to do here.

Reply:
There must be a better way to do this, but using `relative_file` maps a *file* to another file, not a directory, so it needs a file.

It could just be a string slice but that always scares me...

Reply:
Actually if the paths are normalized (🤞) maybe it's fine. See the latest...

Reply:
Now I see a failure though, and doing a plain string slice assumes `src` is always under `root_dir`. I assume it should always be under there but I don't trust myself 🤔

Reply:
Ok, take another look, going to resolve this...